### ISSUE TITLE
initial submission

### ISSUE THREAD

**Issue by MarcoAJanssen** [make-models-fair/nowak-sigmund-1998#1]
I submitted my NetLogo implementation and some model analysis. Most of the results align very well with the published results. Nowak and Sigmund present levels of cooperation for Figures 3 and 4, but do not say for which levels of visibility or kind of runs. My implementation does not find a lower level of cooperation with bigger group numbers. Anybody finding an explanation for this? @Mohsen-Shahbaznezhadfard

**Comment by Mohsen-Shahbaznezhadfard** [comment]
Thank you @MarcoAJanssen !

I did not find any content or article that specifically addresses and scrutinizes this issue with these assumptions.

In a part of the description of Figure 3 of this article, it says that "Each interaction is observed, on average, by ten randomly chosen players." Also, as you know, this analysis is done for three different population sizes, n = 20, n = 50 and n = 100. Isn't this an explanation for the level of visibility?
This article explains about this issue that "For larger groups, it is more difficult to establish cooperation, because the fraction of individuals that obtain information about any particular interaction will be smaller. Therefore, more interactions are required (relative to group size) in order to discriminate against defectors."

**Comment by MarcoAJanssen** [comment]
Good catch on the "ten randomly chosen players". I am redoing the runs, and some sensitivity analysis.

**Comment by MarcoAJanssen** [comment]
The results are updated with different levels of visibility, including the 10 random chosen players - on average. The results are very similar to published results, and the sensitivity analysis shows that visibility is the key factor impacting cooperation, not group size.
A tip is to remove the figures from the Netlogo model to run the large number of ticks without memory problems.

**Comment by Mohsen-Shahbaznezhadfard** [comment]
This is great!
Thank you for your contribution, @MarcoAJanssen !
